core-bundle/src/Cron/PurgeRegistrationsCron.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Contao\CoreBundle\Cron;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\MemberModel;
+use Psr\Log\LoggerInterface;
+
+class PurgeRegistrationsCron
+{
+    public function __construct(private ContaoFramework $framework, private LoggerInterface|null $logger)
+    {
+    }
+
+    public function __invoke(): void
```

Comment:
This PR (and all the other TL_CRON related PRs) still needs the adjustments discussed in https://github.com/contao/contao/pull/4634#discussion_r869090093.